internal/captain/command.go

```diff
@@ -336,6 +342,15 @@ func (c *Command) interceptFunc() InterceptFunc {
 	}
 }
 
+// SetUnstable denotes if the command as a beta feature. This will remove the command
+// from state help, disable the commmand for those who haven't opted in to beta features,
+// and add a warning banner for those who have.
+func (c *Command) SetUnstable(unstable bool) *Command {
+	c.unstable = unstable
+	c.cobra.Hidden = unstable
```

Comment:
This wasn't in the AC's (my bad), but I think if you opted in it should show the command help, along with a warning saying it is unstable.

I've updated the AC's now to reflect this.

Reply:
Updated this to add `(Unstable)` to the end of any unstable commands if the user has opted in. Pinged Luke and Pete incase they want to change how this looks